src/components/ChoiceList/tests/ChoiceList.test.tsx

```diff
@@ -50,6 +50,31 @@ describe('<ChoiceList />', () => {
       });
     });
 
+    it('renders choices with labels containing JSX', () => {
+      const jsxLabel = <b>Two</b>;
+      const ComponentLabel = () => (
+        <Fragment>
```

Comment:
Why do you use `Fragment` here instead of `<>`?  Never used `Fragment` in tests, so curious if it possible at all.

Reply:
`<> ` is syntactic sugar and gets compiled to `<React.Fragment>`. Because of *historical reasons* in Polaris's build we have to use the long-form version and we have an eslint rule to enforce that.

Reply:
I went fancy with that `<Fragment />` but I could simply use a good old `<div />` as well if it better matches your best practices 👍